.github/workflows/nightly-cleanup.yml

```diff
@@ -0,0 +1,21 @@
+name: Nightly Cleanup
+
+on:
+  schedule:
+    # 1am https://crontab.guru/#0_1_*_*_*
+    - cron: "0 1 * * *"
+  workflow_dispatch:
+
+jobs:
+  nightly_cleanup:
+    runs-on: ubuntu-latest
+    environment: Dev
+
+    steps:
+    - name: Clean management-api
+      uses: smartsquaregmbh/delete-old-packages@v0.3.1
+      with:
+        version-pattern: "\\b[0-9a-f]{5,40}\\b" # The regex needs to be escaped!
```

Comment:
nit: comment on example matches and why they are cleaned

Reply:
added.